packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -509,6 +504,73 @@ class PlatformZoomRange {
   final double? max;
 }
 
+class PlatformBitmap {
+  PlatformBitmap({required this.bitmap});
+
+  final Object bitmap;
+}
+
+class PlatformBitmapDefaultMarker {
+  PlatformBitmapDefaultMarker({this.hue});
+
+  final double? hue;
+}
+
+class PlatformBitmapBytes {
+  PlatformBitmapBytes({required this.byteData, this.size});
+
+  final Uint8List byteData;
+  final PlatformOffset? size;
```

Comment:
Either this type needs a more generic name, or there should be a new type for size; a size and an offset are conceptually different things.